src/commands/scheduledQuestion.ts

```diff
@@ -0,0 +1,146 @@
+  },
+  {
+    type: ApplicationCommandOptionType.Subcommand,
+    name: 'view',
+    description: 'See all scheduled question channels you have set up.',
+  },
+] as const;
+
+const scheduledQuestion: Command = {
+  name: 'scheduledquestion',
+  description: 'Automatically post a question every certain amount of time!',
+  options,
+  category: 'control',
+  perms: ['ManageGuild'],
+  run: async (ctx: Context) => {
+    if (!ctx.guildId)
+      return ctx.reply(ctx.client.EMOTES.xmark + ' This command cannot be run in DMs.');
+
+    if (!ctx.premium) return ctx.replyUpsell();
+
+    if (ctx.args[0] === 'create') {
+      if (await ctx.client.database.getScheduledQuestionChannel(ctx.channelId))
+        return ctx.reply(
+          `${ctx.client.EMOTES.xmark} There's already a scheduled question in this channel. Please remove the old one with \`/scheduledquestion remove\` before creating a new one!`
+        );
+
+      const frequency = ctx.getOption<Mutable<typeof options[0]['options'][0]>>('frequency')!.value;
+      let questionType =
+        ctx.getOption<Mutable<typeof options[0]['options'][1]>>('type')?.value || null;
+      let rating = ctx.getOption<Mutable<typeof options[0]['options'][2]>>('rating')?.value || null;
+      const pingRole =
+        ctx.getOption<Mutable<typeof options[0]['options'][3]>>('role')?.value || null;
+
+      if (questionType === 'NONE') questionType = null;
+      if (rating === 'NONE') rating = null;
+
+      await ctx.client.database.createScheduledQuestionChannel({
+        id: ctx.channelId,
+        botId: ctx.client.id,
+        guildId: ctx.guildId,
+        schedule: frequency,
+        type: questionType,
+        rating,
+        role: pingRole,
+      });
+
+      return ctx.reply(
+        `${ctx.client.EMOTES.checkmark} Created a ${frequency.toLowerCase()} scheduled question!`
+      );
+    } else if (ctx.args[0] === 'remove') {
+      await ctx.client.database.deleteScheduledQuestionChannel(ctx.channelId);
+
+      return ctx.reply(
+        `${ctx.client.EMOTES.checkmark} Removed the scheduled question in this channel.`
+      );
+    } else if (ctx.args[0] === 'view') {
+      const scheduledQuestionsForGuild =
+        await ctx.client.database.getGuildScheduledQuestionChannels(ctx.guildId);
+
+      if (!scheduledQuestionsForGuild.length)
+        return ctx.reply(
+          `${ctx.client.EMOTES.xmark} There are no scheduled question channels set up in this server.`
+        );
+
+      ctx.reply({
+        embeds: [
+          {
+            title: `${ctx.client.EMOTES.time} Scheduled Question Channel List`,
+            color: ctx.client.COLORS.BLUE,
+            description: scheduledQuestionsForGuild
+              .map(
+                sc =>
+                  `• **${ctx.client.functions.titleCase(sc.schedule)}:** <#${sc.id}> - ${
+                    sc.type || ''
+                  } ${sc.rating || ''} ${sc.role ? `Pings <@&${sc.role}>` : ''}`
```

Comment:
Rating is not displayed.

<img width="420" alt="CleanShot 2022-12-16 at 13 33 20@2x" src="https://user-images.githubusercontent.com/36977340/208165587-be4502f0-acd8-4f09-bc04-302211ca6374.png">